evergreen.yml

```diff
@@ -362,6 +349,11 @@ functions:
       script: |
         set -eo pipefail
 
+        # Hosts may have incomplete or invalid toolchain directories from
```

Comment:
"3. Put comments in yaml not in inline shell."

Reply:
Can't argue with that!